src/components/pages/GamePlay/GameMissionProgress.js

```diff
@@ -31,37 +31,99 @@ export default function GameMissionProgress(props) {
     }
   };
 
+  // const showModal = () => {
+  //   setIsModalVisible(true);
+  //   const modalData = {
+  //     title: 'Quick Navigation Tips!',
+  //     description:
+  //       'Click on the "Read," "Draw" or "Write" buttons to navigate through the mission. You have to complete your current step before you can move on to the next step, so if you haven\'t completed the "Read" step, you won\'t be able to click on "Draw" or "Write" to advance. Once you have completed all the steps, you can click back and forth between all of them.',
+  //     buttonTxt: 'Got it!',
+  //   };
+
+  //   props.enableModalWindow(modalData);
+  // };
+
+  const showGameTips = () => {
+    return (
+      <div className="game-tips-container">
+        <div className="game-tips-text">
+          <p className="game-tips-text-p">
+            <img
+              style={{ width: '48px', height: '48px' }}
+              src="https://img.icons8.com/flat-round/64/000000/wide-right-arrow.png"
+            />{' '}
+            Click on the "Read," "Draw" or "Write" buttons to navigate through
+            the mission.
+          </p>
+          <p className="game-tips-text-p">
+            <img
+              style={{ width: '48px', height: '48px' }}
+              src="https://img.icons8.com/flat-round/64/000000/wide-right-arrow.png"
+            />{' '}
+            You have to complete your current step before you can move on to the
+            next step
+          </p>
+          <p className="game-tips-text-p">
+            <img
+              style={{ width: '48px', height: '48px' }}
+              src="https://img.icons8.com/flat-round/64/000000/wide-right-arrow.png"
+            />{' '}
+            Once you have completed all the steps, you can click back and forth
+            between all of them.
+          </p>
+        </div>
+        <button className="game-tips-gotit-btn">Got it!</button>
+      </div>
+    );
+  };
+
   return (
-    <div id="mission-progress">
-      <h3>Your Mission</h3>
+    <div>
+      <div>
+        <Dropdown overlay={showGameTips} trigger={['click']}>
+          {/*Lightbulb Icon*/}
+          <div className="game-tips-icon-container">
```

Comment:
Should use a Button component if you want the modal to open on click

Reply:
Also needs ```onClick={(e)=>e.preventDefault()``` as the attribute so that the modal will close when clicking away

Reply:
> Should use a Button component if you want the modal to open on click

The design team agreed that after clicking the icon, the drop-down info box that will appear should have the same design as the drop-down hamburger menu so I chose to use ```<Dropdown/>``` tag as in the Header.js file.